core/src/main/java/com/microsoft/applicationinsights/internal/agent/CoreAgentNotificationsHandler.java

```diff
@@ -142,54 +142,35 @@ public String getName() {
     }
 
     @Override
-    public void httpMethodFinished(String identifier, String method, String correlationId, String uri, String target, int result, long deltaInNS) {
+    public void httpMethodFinished(String identifier, String method, String correlationId, String uri, String target, int result, long deltaInNS)
+		throws URISyntaxException {
         if (!LocalStringsUtils.isNullOrEmpty(uri) && (uri.startsWith("https://dc.services.visualstudio.com") || uri.startsWith("https://rt.services.visualstudio.com"))) {
             return;
         }
         long deltaInMS = nanoToMilliseconds(deltaInNS);
-        RemoteDependencyTelemetry telemetry = new RemoteDependencyTelemetry(identifier, null, new Duration(deltaInMS), true);
-        telemetry.setId(correlationId);
-        telemetry.setResultCode(Integer.toString(result));
-        telemetry.setType("HTTP");
-        telemetry.getContext().getProperties().put("URI", uri);
```

Comment:
@dhaval24, I think we should add these context properties into the other method. This could break a customer query which used these properties in the previous version.

What do you think, @grlima ?

Reply:
@littleaj thanks I have just fixed this